tomb-common/Cargo.toml

```diff
@@ -30,11 +30,12 @@ sha1 = "^0.10"
 sha2 = "^0.10"
 test-case = "^3"
 thiserror = "^1"
-uuid = { version = "^1", features = ["serde"] }
+uuid = { workspace = true }
```

Comment:
We do not want to be doing this workspace thing. Using dependencies inside workspaces already share the dependencies when they naturally overlap. It does not enforce the same versioning for dependencies of our dependencies and makes it harder to update individual crates.